### PR TITLE
[Storage] Fix parameter descriptions of Set-AzStorageLocalUser

### PR DESCRIPTION
**Pull request by yifanz0**
## Description
Update parameter descriptions per issue https://github.com/Azure/azure-powershell/issues/21020

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Storage/Storage.Management/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Updated parameter descriptions for Set-AzStorageLocalUser
+    - `Set-AzStorageLocalUser`
```
Comment (BethanyZhou) [inline src/Storage/Storage.Management/ChangeLog.md:22]: No change log is needed for doc changes.

Comment (yifanz0) [reply src/Storage/Storage.Management/ChangeLog.md]: Fixed

**Review by BethanyZhou — DISMISSED** [review]

File: src/Storage/Storage.Management/StorageAccount/SetAzureStorageLocalUser.cs [hunk 2]
```diff
@@ -68,12 +68,12 @@ public class SetAzureStorageLocalUserCommand : StorageAccountBaseCmdlet
...
 
         [Parameter(Mandatory = false,
-            HelpMessage = "The name of local user. The username must contain lowercase letters and numbers only. It must be unique only within the storage account.")]
+            HelpMessage = "Local user ssh authorized keys for SFTP.")]
         [ValidateNotNullOrEmpty]
         public PSSshPublicKey[] SshAuthorizedKey { get; set; }
 
         [Parameter(Mandatory = false,
-            HelpMessage = "The name of local user. The username must contain lowercase letters and numbers only. It must be unique only within the storage account.")]
+            HelpMessage = "The permission scopes of the local user.")]
```
Comment (blueww) [inline src/Storage/Storage.Management/StorageAccount/SetAzureStorageLocalUser.cs:76]: I would suggest also mention how to create the parameter. 
refer the sample: https://learn.microsoft.com/en-us/powershell/module/az.storage/set-azstorageblobinventorypolicy?view=azps-9.4.0#-rule

It should be created with cmdlet : New-AzStorageLocalUserPermissionScope

**Review by blueww — CHANGES_REQUESTED** [review]

File: src/Storage/Storage.Management/StorageAccount/SetAzureStorageLocalUser.cs (hunk 2 above)
Comment (yifanz0) [reply src/Storage/Storage.Management/StorageAccount/SetAzureStorageLocalUser.cs]: Done

**Review by blueww — APPROVED** [review]

**Pull request merged**